extension/landing-zone/variables.tf

```diff
@@ -0,0 +1,328 @@
+variable "ibmcloud_api_key" {
+  description = "The API key that's associated with the account to provision resources to"
+  type        = string
+  sensitive   = true
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example."
```

Comment:
Update the description here. I guess this needs to be the resource group that the SLZ VPC is in?